adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -604,13 +604,13 @@ def _determine_color(
             if self._generator.ontology.is_subtype_of(property_, COLOR)
         )
 
-        prototypical_colors_for_object_type = immutableset(
-            property_
-            for property_ in self._generator.ontology.properties_for_node(
-                situation_object.ontology_node
-            )
-            if self._generator.ontology.is_subtype_of(property_, COLOR)
-        )
+        # prototypical_colors_for_object_type = immutableset(
```

Comment:
Commenting out these blocks of code prevented the root object from getting re-assigned a random color. Would it be safe to remove these (and let the "prototypical" colors be determined within `_perceive_objects`)?